### PR TITLE
Add flake8 and black configuration

### PR DESCRIPTION
**Pull request by joeleu**
This PR adds configuration files for both flake8 and black. Those files are then set in the `.vscode/settings.json` to ensure the editor and its plugins use the same configuration. This makes sure both the CI/CD environment and our local environments have the same linting and formatting expectations.

Finally, the Python related changes are due to these changes in the configuration and are formatting only.

### CODE REVIEW

File: voluncheer/.flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 100
+exclude = */migrations/*
```
Comment (JinweiZhu) [inline voluncheer/.flake8:3]: *_initial.py should also be included? Those migration files are generated by the system.

**Review by JinweiZhu — CHANGES_REQUESTED** [review]
Minor change is needed but otherwise looks good.

File: voluncheer/.flake8 (hunk 1 above)
Comment (joeleu) [reply voluncheer/.flake8:3]: The only `*_initial.py` files I saw were within a `migrations/` directory so they are covered by this exclude. Do these files appear elsewhere?

Comment (JinweiZhu) [reply voluncheer/.flake8:3]: Good point.

**Review by JinweiZhu — APPROVED** [review]

**Pull request merged**